src/business/dal/mappers/data-mappers.factory.ts

```diff
@@ -1,40 +1,58 @@
-import FirebaseAdapter from '@business/dal/firebase/firebase.adapter';
-import PartyEventMapper from '@business/modules/party-event/party-event.mapper';
 import CalculationModificationMapper from '@business/modules/calculation-modification/calculation-modification.mapper';
 import ExcludeModificationMapper from '@business/modules/exclude-modification/exclude-modification.mapper';
+import PartyEventMapper from '@business/modules/party-event/party-event.mapper';
 import PaymentMapper from '@business/modules/payment/payment.mapper';
-import UserMapper from '@business/modules/user/user.mapper';
-import { FirebaseEntitiesList } from '@business/dal/firebase/firebase-entities.list';
 import UserEventPropertiesMapper from '@business/modules/user-event-properties/user-event-properties.mapper';
+import UserMapper from '@business/modules/user/user.mapper';
+import AdaptersFactory, { AdapterType } from '../adapters/adapters.factory';
+import { Entity } from '../adapters/entities.list';
 
 export default class DataMappersFactory {
-  public createPartyEventMapper(): PartyEventMapper {
-    const adapter = new FirebaseAdapter(FirebaseEntitiesList.partyEvent);
-    return new PartyEventMapper(adapter);
+  private _adaptersFactory: AdaptersFactory;
+
+  constructor() {
+    this._adaptersFactory = new AdaptersFactory();
+  }
+
+  public createPartyEventMapper(type = AdapterType.API): PartyEventMapper {
+    return new PartyEventMapper(
```

Comment:
### 🤓 Nitpick: It will be readable if we do it in 2 steps

    const adapter =  this._adaptersFactory.createAdapter(type, Entity.PARTY_EVENT);
    return new PartyEventMapper(adapter);